### DISCUSSION TITLE
Node flow into a community (continuous community membership)?

### DISCUSSION THREAD

**Discussion by kochbj** [mapequation/infomap#179] [Q&A] [answered]
Hi guys,

Thanks so much for maintaining InfoMap as such a polished product. I'm interested in getting a continuous metric for node membership to communities in a single layer network. I was thinking of looking at the proportion of flow for a node into a module/through constituents of that module. Is there a way to access this in the API? Alternatively, do you think there might be a more appropriate measure of continuous community membership?

Thanks for your time; I really appreciate it.

-Bernie

**Comment by antoneri** [comment, marked as answer]
Hi and thanks for reaching out!

For ordinary networks, Infomap assigns nodes to hard partitions. One way to measure significant assignments is through resampling (https://www.mapequation.org/publications.html#Rosvall-Bergstrom-2010-Mapping-change). 

Maybe @danieledler has some ideas about this?

Best,
Anton
